nix/test/basic/fio_nvme_basic.nix

```diff
@@ -0,0 +1,109 @@
+{ pkgs, lib, ... }:
+let
+  targetIp = "192.168.0.1";
+  initiatorIp = "192.168.0.2";
```

Comment:
Doesn't a fixed ip address limit where this test can be deployed?

Reply:
Ideally, we don't have fixed IPs however, mayastor needs to have an IP in order to run and we don't have code that can "auto-discover" the IPs (as they are passed in as ENV through k8s).